src/AnnotationReader.php

```diff
@@ -82,7 +82,7 @@ public function getClassAnnotations(ReflectionClass $class) : array
     /**
      * {@inheritDoc}
      */
-    public function getClassAnnotation(ReflectionClass $class, $annotationName)
+    public function getClassAnnotation(ReflectionClass $class, string $annotationName)
```

Comment:
I would suggest to split decorative CS changes and BC-breaking signature changes into separate distinct commits.
Adding a type hint changes the signature, which breaks any implementing classes.

Reply:
And then, the decorative code style changes can go into a dedicated PR to the 1.x branch.